docs/builtins/General/read.md

```diff
@@ -1,32 +1,39 @@
 ## read
-Use the `read` function to retrieve information from a specified `table` based on a given `key`. This function returns the database record object corresponding to the `key`. Optionally, you can specify a list of `columns` to retrieve specific data from the record.
+
+Use the `read` function to retrieve information from a specified `table` based on a given `key`. 
+By default, this function returns the complete database record for the specified `key`. 
+Optionally, you can specify one or more `columns` to retrieve specific data from the record.
 
 ### Basic syntax
 
 To retrieve the entire record associated with the specified `key`, use the following syntax:
 
-`(read table key)`
+```pact
+(read table key)
+```
 
 To retrieve specific columns from the record associated with the specified `key`, use the following syntax:
 
-`(read table key [columns])`
+```pact
+(read table key [columns])
+```
 
 ### Arguments
 
 Use the following arguments to specify the table and key for retrieving data using the `read` Pact function.
 
 | Argument | Type | Description |
 | --- | --- | --- |
-| `table` | `table:<{row}>` | Specifies the table from which to retrieve data. |
-| `key` | `string` | Specifies the unique identifier (key) for the record to retrieve. |
+| `table` | table:<{row}> | Specifies the table from which to retrieve data. |
+| `key` | string | Specifies the unique identifier (key) for the record to retrieve. |
 
 ### Options
 
 Use the following option to specify particular columns to retrieve from the record associated with the specified `key` in the `read` Pact function.
 
 | Option | Type | Description |
 | --- | --- | --- |
-| `columns` | `[string]` | Specifies the list of columns to retrieve from the record. |
+| `columns` | [string] | Specifies the list of columns to retrieve from the record. |
```

Comment:
We should align on how to represent polymorphic types. i.e., is it a `list`, or `[<a>]`, ...